lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -812,6 +813,20 @@ public void checkCanGrantExecuteFunctionPrivilege(SystemSecurityContext context,
     {
     }
 
+    @Override
+    public void checkCanGrantExecuteFunctionPrivilege(SystemSecurityContext context, FunctionKind functionKind, CatalogSchemaRoutineName functionName, TrinoPrincipal grantee, boolean grantOption)
+    {
+        switch (functionKind) {
+            case SCALAR, AGGREGATE, WINDOW:
+                return;
+            case TABLE:
+                // TODO implement
```

Comment:
I guess it's unnecessary comment. 
Even more, this has to be implemented for `SCALAR, AGGREGATE, WINDOW` too.

Reply:
> I guess it's unnecessary comment.

it conveys this is temporary

> Even more, this has to be implemented for SCALAR, AGGREGATE, WINDOW too.

it doesn't have to be.

Reply:
> It's not consistent and makes to think that it's temporary here, but not temporary in other places

i didn't add those other places. I am conveying intentions as they are current.

> It's even true for TABLE 😉

We believe TABLE functions require different treatment, as they are considered "not safe by default". So we **must** have access control rules to allow users use them.

OTOH, scalar, aggregate and window functions are considered "safe by default", so we don't have to have access control rules. **Maybe** someone someday will have a use-case where it's needed, but also **maybe** it never happens and we never implement such rules.